### ISSUE TITLE
Successful implementations?

### ISSUE THREAD

**Issue by Stafla** [deepsound-project/samplernn-pytorch#31]
Hi, 

Is there anyone who successfully generated audio using this implementation of SampleRNN? If so, what parameter set did you use? What was your final training and validation loss? Furthermore, did you adjust any code in order to produce audio of good quality?

I ran several instances of this implementation to learn pattern in classical music, using different parameters. The lowest training  and validation loss I retrieved was 0.72 and 0.86 (NLL in bits), respectively. The corresponding parameter set was equivalent to the optimal music parameters set according to Mehri et al. However, the generated audio files are still noisy. I know there are people who generated qualitative audio files using the original Theano implementation, although their losses were higher. I, therefore, am wondering if there are people who have or had similar issues. Your help is appreciated :).

Kind regards, 

Stafla

**Comment by dmtrs** [comment]
Hey @Stafla, did you managed to run other implementations? If so, how would compare them with this one?

Could you please share audion samples from your tests?
